config/roguelike_dungeons/settings/loot_reward.json

```diff
@@ -0,0 +1,86 @@
+		{"data": {"name": "IronChest:BlockIronChest", "meta": 6}, "weight": 10},
+		{"data": {"name": "IC2:itemArmorAlloyChestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "TConstruct:materials", "meta": 26}, "weight": 10},
+		{"data": {"name": "ExtraUtilities:spike_base_diamond", "meta": 0}, "weight": 10},
+		{"data": {"name": "ExtraUtilities:trading_post", "meta": 0}, "weight": 10},
+		{"data": {"name": "ArchitectureCraft:sawbench", "meta": 0}, "weight": 10},
+		{"data": {"name": "OpenBlocks:hangglider", "meta": 0}, "weight": 10},
+		{"data": {"name": "Backpack:workbenchbackpack", "meta": 0}, "weight": 10},
+		{"data": {"name": "naturescompass:NaturesCompass", "meta": 0}, "weight": 10},
+		{"data": {"name": "gregtech:gt.blockmachines", "meta": 9250}, "weight": 5},
+		{"data": {"name": "gregtech:gt.blockmachines", "meta": 9330}, "weight": 5}
+    ]
+    },
+	{"level": 2, "type": "REWARD", "each": true, "quantity": 1, "loot": [
+		{"data": {"name": "adventurebackpack:adventureBackpack", "meta": 24}, "weight": 10},
+		{"data": {"name": "BuildCraft|Factory:mover", "meta": 0}, "weight": 10},
+		{"data": {"name": "etfuturum:totem_of_undying", "meta": 0}, "weight": 10},
+		{"data": {"name": "IronChest:BlockIronChest", "meta": 2}, "weight": 10},
+		{"data": {"name": "gregtech:gt.metaitem.01", "meta": 32727}, "weight": 10},
+		{"data": {"name": "TConstruct:materials", "meta": 8}, "weight": 10},
+		{"data": {"name": "malisisdoors:door_factory", "meta": 0}, "weight": 10},
+		{"data": {"name": "OpenBlocks:guide", "meta": 0}, "weight": 10},
+		{"data": {"name": "ExtraTrees:gt.machine", "meta": 1}, "weight": 3},
+		{"data": {"name": "ExtraTrees:gt.machine", "meta": 2}, "weight": 3},
+		{"data": {"name": "ExtraTrees:gt.machine", "meta": 3}, "weight": 2},
+		{"data": {"name": "ExtraTrees:gt.machine", "meta": 4}, "weight": 2}
+      ]
+    },
+	{"level": 3, "type": "REWARD", "each": true, "quantity": 1, "loot": [
+		{"data": {"name": "TConstruct:materials", "meta": 6}, "weight": 10},
+		{"data": {"name": "Avaritia:Ultimate_Stew", "meta": 0}, "weight": 10},
+		{"data": {"name": "BinnieCore:storage", "meta": 3}, "weight": 10},
+		{"data": {"name": "ExtraUtilities:chandelier", "meta": 0, "min": 5, "max": 8}, "weight": 10},
+		{"data": {"name": "Forestry:beealyzer", "meta": 0}, "weight": 3},
+		{"data": {"name": "Forestry:flutterlyzer", "meta": 0}, "weight": 3},
+		{"data": {"name": "Forestry:treealyzer", "meta": 0}, "weight": 3},
+		{"data": {"name": "MagicBees:frameOblivion", "meta": 0}, "weight": 10},
+		{"data": {"name": "OpenBlocks:builder_guide", "meta": 0}, "weight": 10},
+		{"data": {"name": "ThaumicHorizons:boatThaumium", "meta": 0}, "weight": 10},
+		{"data": {"name": "OpenBlocks:elevator", "meta": 0, "min": 2, "max": 3}, "weight": 10},
+		{"data": {"name": "Thaumcraft:blockMetalDevice", "meta": 6, "min": 6, "max": 8}, "weight": 10}
+    ]
+    },
+    
+	{"level": 4, "type": "REWARD", "each": true, "quantity": 1, "loot": [
```

Comment:
I see you're defining level 4 reward pools but there is no reward room on level 4 in any of the room configurations you've updated. Is this because the reward pool is already given out in the existing final room that has 4 chests, and so players would get the level 4 reward pool 4 times per dungeon?